From 57fb77e6bdfade0703f1c422279ad4353edbe980 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Fri, 14 Sep 2018 13:18:14 +0800
Subject: [PATCH 4644/5242] MLK-19581-5: ARM64: dts: support ak4497 over rpmsg
 in default

commit  94abece15a1b67c4f7a47a583a9d05c576cfe6ef from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable LPA in default.
Change the module name, for alsactl can restore the default
setting by name

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit 6827c27c07e5563a11d0b77456672d05ed5db650)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
index 1f9c667..6490389 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
@@ -40,12 +40,13 @@
 		/* the audio device index in m4 domain */
 		fsl,audioindex = <0> ;
 		fsl,dma-buffer-size = <0x6000000>;
+		fsl,enable-lpa;
 		status = "okay";
 	};
 
 	sound-rpmsg {
 		compatible = "fsl,imx-audio-rpmsg";
-		model = "rpmsg-audio";
+		model = "ak4497-audio";
 		cpu-dai = <&rpmsg_i2s>;
 		rpmsg-out;
 	};
-- 
1.7.9.5

